apps/syntrest/src/main/java/no/nav/registre/syntrest/controllers/SyntController.java

```diff
@@ -337,21 +300,22 @@
         return ResponseEntity.ok(response);
     }
 
-    @PostMapping("/elsam")
-    @ApiOperation(value = "Generer sykemelding-historie", notes = "Lager en sykemeldings-historie per person som blir sendt inn. ")
-    @Timed(value = "syntrest.resource.latency", extraTags = {"operation", "synthdata-elsam"})
-    public ResponseEntity<Map<String, Historikk>> generateSykemeldingHistorikk(
-            @ApiParam(value = "Fnr for personen som skal ha sykemeldinger. Startdato for sykemeldingene.", required = true, example = "")
-            @RequestBody Map<String, String> fnrStartdatoMap
-    ) {
-        InputValidator.validateInput(new ArrayList<>(fnrStartdatoMap.keySet()));
-
-        Map<String, Historikk> response = (Map<String, Historikk>)
-                elsamConsumer.synthesizeData(UriExpander.createRequestEntity(elsamUrl, fnrStartdatoMap));
-        doResponseValidation(response);
 
-        return ResponseEntity.ok(response);
-    }
+//    @PostMapping("/elsam")
```

Comment:
Synses selv om vi kan få bruk for det en dag så brude kommentert ut kode slettes. Du kan finne det igjen i historikken

Reply:
Den er grei, jeg skal fjerne :)